roslibrust_test/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+use roslibrust_codegen_macro::find_and_generate_ros_messages;
+
+find_and_generate_ros_messages!("..");
```

Comment:
I'm curious about how often this gets evaluated / if we prefer to have a static version here.

Reply:
Hmm good point that I hadn't considered. There's not really any way for `cargo` to know this crate has dependencies on the message files and should re-compile the crate if one of those message files changes. This is a possible footgun. I'm not sure if that warrants taking it out or if it should just get a lot of explicit documentation around when it will/won't recompile.